src/cmd/builtin.py

```diff
@@ -1582,3 +1584,27 @@ async def _reloadbotcommands(message, command, silent=False):
         await Msg.response(message, "Bot commands reloading is started...", silent)
         bc.commands.update(reload=True)
         await Msg.response(message, "Bot commands reloading is finished", silent)
+
+    @staticmethod
+    async def _permlevel(message, command, silent=False):
+        """Get permission level for current user
+    Usage: !permlevel
+           !permlevel `@user`"""
+        if not await Util.check_args_count(message, command, silent, min=1, max=2):
+            return
+        info = ""
+        user_id = 0
+        if len(command) == 1:
+            info = message.author
+            user_id = message.author.id
+        elif len(command) == 2:
+            if not message.mentions:
+                await Msg.response(message, "You need to mention the user you want to get profile of", silent)
+                return
+            info = await message.guild.fetch_member(message.mentions[0].id)
+            user_id = message.mentions[0].id
+        if info is None or user_id not in bc.config.users.keys():
+            await Msg.response(message, "Could not get permission level for this user", silent)
+            return
+        perm_level = bc.config.users[user_id].permission_level
+        await Msg.response(message, f"Permission level for {f'{info.nick} ({info})'if info.nick is not None else info} is {perm_level}", silent)
```

Comment:
Since this command can be used as subcommand it should also return result string. So it should look like this:
```py
    ...
    result = f"Permission level for {f'{info.nick} ({info})'if info.nick is not None else info} is {perm_level}"
    await Msg.response(message, result, silent)
    return result
```

Reply:
In case of command failure empty return is okay